docs/source/tutorials/configure_systems.rst

```diff
@@ -101,6 +101,25 @@ for :code:`"conda.yaml"`.
 
 Please refer to :ref:`azureml_system_config` for more details on the config options.
 
+AzureML Readymade Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are some readymade systems available for AzureML. These systems are pre-configured with the necessary.
+    .. code-block:: json
+
+            {
+                "type": "AzureNDV2System",
```

Comment:
Nit: Shall we link the detailed system alias to azure page? Just like:
AzureNV2System -> https://learn.microsoft.com/en-us/azure/virtual-machines/ndv2-series

Reply:
there are other skus so I don't put the link in the document. I put them in the python comments.